app/matplotlib.ipynb

```diff
@@ -86,23 +90,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
     "x = np.random.rand(50)\n",
     "y = np.random.rand(50)\n",
     "colors = np.random.rand(50)\n",
     "sizes = 100 * np.random.rand(50)\n",
     "\n",
-    "# write your code here"
+    "plt.scatter(x, y, c=colors, s=sizes)\n",
+    "plt.title(\"Random colors and sizes\")\n",
+    "\n",
+    "plt.xlabel(\"x\")\n",
+    "plt.ylabel(\"y\")"
```

Comment:
You need to add `plt.colorbar()` to include the colorbar in the scatter plot, and `plt.show()` to display the plot.